meta-mender-core/recipes-mender/mender-connect/mender-connect_2.0.0.bb

```diff
@@ -0,0 +1,30 @@
+require mender-connect.inc
+
+RDEPENDS_${PN} = "glib-2.0 mender-client (>= 3.2)"
```

Comment:
I think it is better to remove this line (as suggested before, maybe wasn't clear?). `RDEPENDS_` is already defined in `mender-connect.inc` and does not need to be overridden (unlike the 1.x recipes that need to override the version of mender-client to depend on).

Reply:
Fixed